tests/cluster/tests/04-resharding.tcl

```diff
@@ -84,7 +84,7 @@ test "Cluster consistency during live resharding" {
             flush stdout
             set target [dict get [get_myself [randomInt 5]] id]
             set tribpid [lindex [exec \
-                ../../../src/valkey-cli --cluster reshard \
+                ../../../src/valkey-cli -c --cluster reshard \
```

Comment:
Can't we make `--cluster` imply `-c`?

It looks redundant/silly to pass both these options.

Reply:
Maybe, but the priority now is to fix the build. I don't think this should be changed in this PR. We need to understand why '-c' was added in the first place.

Reply:
> Maybe, but the priority now is to fix the build.

Yes, definitely.

> We need to understand why '-c' was added in the first place.

AFAIK:

* `-c` is for following redirects when calling simple commands like `valkey-cli -c get foo`.
* `--cluster` is for special cluster admin operations.